### DISCUSSION TITLE
Relationship to Azure Health Data Services DICOM service

### DISCUSSION THREAD

**Discussion by mekinney** [microsoft/dicom-server#1454] [General]
Is this dicom-server solution related to the Azure Health Data Services DICOM service in any way?  I'm hoping there's a doc somewhere that describes the differences / relationship.

**Comment by mekinney** [comment]
Answering my own question - https://docs.microsoft.com/en-us/azure/healthcare-apis/dicom/references-for-dicom-service

**Comment by smithago** [comment]
That is right. I also have a doc on what to consider if you use OSS, which are solved in PaaS.  https://github.com/microsoft/dicom-server/blob/main/docs/resources/dicom-server-maintaince-guide.md
